docs/migration/gcloud/vector.md

```diff
@@ -0,0 +1,162 @@
+# Vector
+
+*Migrating from Google Cloud to Siglens using Vector*
+
+## 1. Install Vector and setup Google Cloud CLI
+
+Prerequisites: 
+- Google Cloud CLI (Verify your Google Cloud CLI installation with `gcloud version` or you can install it from [here](https://cloud.google.com/sdk/docs/install-sdk))
+- Vector (You can follow [this guide](../../log-ingestion/vector.md#1-installation) to get started with installing vector)
+<details>
+<summary>Setup Google Cloud Pub/Sub</summary>
+
+#### 1. Set or Create a Google Cloud Project
+
+If you already have a Google Cloud project, set it as your current project:
```

Comment:
I think this whole `<details>` part makes more sense without the `<details>` tags so that it isn't collapsed; that way the reader can follow along with all the steps. Or did you do it this way so if they already have a Google Cloud project setup, they could skip this step and go on to step 2? In that case, they would still need to find their `project_id` and `subscription_name`, right?